<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add Shielding to Base Spectra</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="gboxInput">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4" rowstretch="1,10">
        <item row="1" column="0">
         <widget class="QGroupBox" name="gboxBaseSpec">
          <property name="title">
           <string>Base Spectra</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QListView" name="lstBaseSpectra">
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="hlayoutDetector" stretch="1,2">
          <item>
           <widget class="QLabel" name="labelDetector">
            <property name="text">
             <string>Detector</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbDetectors"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gboxGenParams">
       <property name="title">
        <string>Shield generation parameters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="1">
         <widget class="QComboBox" name="cmbDrf"/>
        </item>
        <item row="6" column="0" colspan="2">
         <widget class="QGroupBox" name="gboxThickness">
          <property name="title">
           <string>Available Thicknesses</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>4</number>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QListView" name="lstThickness">
             <property name="selectionMode">
              <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QComboBox" name="cmbShield"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="labelDrf">
          <property name="text">
           <string>DRF</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="btnAddConfig">
          <property name="text">
           <string>Add Configurations</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QPushButton" name="btnBrowseConfig">
          <property name="text">
           <string>Change Configuration File</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="labelShielding">
          <property name="text">
           <string>Shielding</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtConfigPath">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelConfigPath">
          <property name="text">
           <string>Config File</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gboxGenSpec">
       <property name="title">
        <string>Shielded spectra to be generated</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QListView" name="lstOutSpecs">
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnRemoveConfigs">
          <property name="text">
           <string>Remove Configurations</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxChNum">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string># Channels</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="txtChNum">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBoxEcal">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Energy Calibration (from base spectra)</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="labelEcaltotE">
          <property name="text">
           <string>E=</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal1</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtEcal0">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEcal0">
          <property name="text">
           <string>+</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal2</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtEcal1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEcal1">
          <property name="text">
           <string>x +</string>
          </property>
          <property name="buddy">
           <cstring>txtEcal3</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtEcal2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEcal2">
          <property name="text">
           <string>x^2 +</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtEcal3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="frame">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelEcal3">
          <property name="text">
           <string>x^3</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
